i18n/en-US.yml

```diff
@@ -154,6 +154,7 @@ common:
     tripDurationFormat: >-
       {hours, plural, =0 {} other {# hr }}{minutes} min { seconds, plural, =0 {}
       other {# sec}}
+  linkOpensNewWindow: "(Opens new window)"
```

Comment:
I thought maybe it would be nice for both sighted and AT users to have this broken up a bit with punctuation! Otherwise you get lines like "go to agency site opens new window" which feels a bit rough.